resharper/src/resharper-unity/ProjectExtensions.cs

```diff
@@ -28,6 +32,12 @@ public static class ProjectExtensions
             ourUnityEditorReferenceName, ourUnityEngineReferenceName, ourUnityEngineCoreModuleReferenceName, ourUnityEngineSharedInternalsModuleReferenceName
         };
 
+        public static bool IsUnitySolution([NotNull] this ISolution solution)
+        {
```

Comment:
Maybe an assert here for solution load being fully done?

Reply:
I think 'false' in that case is OK =)